package.json

```diff
@@ -11,23 +11,29 @@
     "clean": "turbo clean",
     "format": "prettier --write .",
     "format-check": "prettier --check .",
-    "all-check": "pnpm install && pnpm format && pnpm format-check && pnpm lint && pnpm test && pnpm build"
```

Comment:
Was this an intentional removal? My understanding is that `pnpm format` will only auto-fix the things it can, but it might not be able to auto-fix everything — in which case, `pnpm format-check` is still useful to keep here.